test/operation/model_test.rb

```diff
@@ -19,11 +19,8 @@ class Create < Trailblazer::Operation
   end
 
   # :new new.
-  it { Create.(params: {})[:model].inspect.must_equal %{#<struct ModelTest::Song id=nil, title=nil>} }
-  it do
-
-    result = Create.(params: {})
-
+  it "initializes new model's instance - even without `params` parameter present in an operation" do
```

Comment:
I guess we don't need to mention `even without params` in description, just simple model initialzation. We won't be needing reference about it in future, do we ?

Reply:
sure, I'll change :+1: